airflow/serialization/serialized_objects.py

```diff
@@ -1195,7 +1195,7 @@ def serialize_mapped_operator(cls, op: MappedOperator) -> dict[str, Any]:
         if TYPE_CHECKING:  # Let Mypy check the input type for us!
             _ExpandInputRef.validate_expand_input_value(expansion_kwargs.value)
         serialized_op[op._expand_input_attr] = {
-            "type": get_map_type_key(expansion_kwargs),
+            "type": type(expansion_kwargs).EXPAND_INPUT_TYPE,
```

Comment:
```suggestion
            "type": expansion_kwargs.EXPAND_INPUT_TYPE,
```

Is the `type` call required? It’s generally not since Python exposes class-level variables in instances automatically. Not sure if I’m missing some magic here.

Reply:
There must have been a reason I did this, but I can't for the life of me think what it was now.